app/Http/Controllers/API/SuggestController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\Building;
+use App\Models\Architect;
+
+class SuggestController extends Controller
+{
+
+    public function buildings(Request $request)
+    {
+        $buildings = Building::searchRaw([
+            'suggest' => [
+                'buildings_1' => [
+                    'prefix' => $request->get('search'),
+                    'completion' => [
+                        'field' => 'title.suggest'
+                    ]
+                ],
+                'buildings_2' => [
+                    'prefix' => $request->get('search'),
+                    'completion' => [
+                        'field' => 'title_alternatives.suggest'
+                    ]
+                ]
+            ]
+        ]);
+
+        $ids_1 = array_column($buildings['suggest']['buildings_1'][0]['options'], '_id');
+        $ids_2 = array_column($buildings['suggest']['buildings_2'][0]['options'], '_id');
+        $buildings = Building::findMany(array_merge($ids_1, $ids_2));
+
+        return \App\Http\Resources\Building::collection($buildings);
```

Comment:
Not sure whether we actually need the resource here, especially since we've dropped the count now. Wouldn't this be enough?
```suggestion
        return $buildings->map->only('id', 'url', 'architect_names', 'title')
```

`architects` field would be re-named to `architect_names`, but I think that would be a simple change to accommodate in JS.